src/App.vue

```diff
@@ -2,20 +2,25 @@
   <div id="app">
     <div
       ref="wrapper"
-      class="min-h-screen wrapper"
-    >
```

Comment:
though not an error, but linter complains - `>` should be on a new line if attributes are multilined
```
> vue-cli-service lint --no-fix
warning: Expected 1 line break before closing bracket, but no line breaks found (vue/html-closing-bracket-newline) at src/App.vue:5:35:
  3 |     <div
  4 |       ref="wrapper"
> 5 |       class="min-h-screen wrapper">
    |                                   ^
  6 |       <div
  7 |         v-if="!isSupportedBrowser"
  8 |         style="text-align: center;line-height: 5rem;">
warning: Expected 1 line break before closing bracket, but no line breaks found (vue/html-closing-bracket-newline) at src/App.vue:8:54:
   6 |       <div
   7 |         v-if="!isSupportedBrowser"
>  8 |         style="text-align: center;line-height: 5rem;">
     |                                                      ^
   9 |         Unsupported browser.
  10 |       </div>
  11 |       <router-view />